### DISCUSSION TITLE
Commonly_used_KeyboardShortcut_in_VSCode(Windows)

### DISCUSSION THREAD

**Discussion by WEI-CHEN-CHUN** [1101-GS4719-C/Python-Community-GS4719-C#746] [General]
"""習慣用快捷鍵，寫程式幾乎不需要滑鼠.."""
VS Code的優點是很多快捷鍵，就算本身沒有，也可以自己新增，或是預設不好用，可以自己改。
推薦大家可以登陸VS Code Account，就算換了裝置，設定都會保留

官方給的快捷鍵總覽：
[(Defaulted)keyboard-shortcuts-windows.pdf](https://github.com/NCU-GS4719-Python/Python-Community/files/9714068/Defaulted.keyboard-shortcuts-windows.pdf)
我寫程式時整理出覺得常用(部分是我自己設定的，如果要用要自己去改喔)：
[Commonly_used_KeyboardShortcut_in_VSCode.txt](https://github.com/NCU-GS4719-Python/Python-Community/files/9714069/Commonly_used_KeyboardShortcut_in_VSCode.txt)
^建議大家使用VS Code 開啟這個檔案

**Comment by dabbie109303501** [comment]
快捷鍵真的蠻好用的，感謝分享~
btw 改快捷鍵的地方在這裡(按右鍵，然後找到鍵盤快速鍵)
![image](https://user-images.githubusercontent.com/75252877/194038457-6e1304aa-5398-4272-b629-a21b334be034.png)

**Reply by WEI-CHEN-CHUN** [reply]
改快捷鍵會用的快捷鍵
Ctrl+K+S(開啟快捷鍵設定的面板)
Alt+K(切換搜尋模式)

**Comment by Zoe06060606** [comment]
快捷鍵真的很方便
不用滑鼠真的超快樂

**Comment by dinosaur1009** [comment]
快捷鍵真的加快寫程式的速度超多，可以加速整個過程

**Comment by Rodrigo-dev-bit** [comment]
vscode的shortcut真的會讓你省時間，尤其是改名稱，可以一次全部改好。
![image](https://user-images.githubusercontent.com/55293456/195099585-b31cd53e-886b-47c5-b671-d4da11da1744.png)

**Comment by williamyen0623** [comment]
再次讚嘆vscode的強大！

**Comment by danwan1027** [comment]
這就是為什麼 vscode 是最受歡迎的編輯器
